about.html

```diff
@@ -0,0 +1,175 @@
+
+    gtag("config", "G-0JED4GM2T0");
+  </script>
+  <title>HTML-CSS-JAVASCRIPT CAPSTONE</title>
+  <!-- Bootstrap CSS -->
+  <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/css/bootstrap.min.css" rel="stylesheet"
+    integrity="sha384-EVSTQN3/azprG1Anm3QDgpJLIm9Nao0Yz1ztcQTwFspd3yD65VohhpuuCOmLASjC" crossorigin="anonymous">
+  <link rel="stylesheet" type="text/css" href="styles.css">
+  <link rel="stylesheet" href="https://pro.fontawesome.com/releases/v5.10.0/css/all.css" integrity="sha384-AYmEC3Yw5cVb3ZcuHtOA93w35dYTsvhLPVnYs9eStHfGJvOvKxVfELGroGkvsg+p" crossorigin="anonymous"/>
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+  <link href="https://fonts.googleapis.com/css2?family=Lato:wght@300;400;700&display=swap" rel="stylesheet">
+</head>
+<body>
+
+  <!-- Navigation Bar Section -->
+
+  <nav class="container-fluid d-none d-md-flex flex-column fixed-top desktop-nav">
+    <div class="d-flex justify-content-end top-nav-bar py-1 px-5">
+      <div>
+          <a href="https://www.facebook.com/"><i class="fab fa-facebook-f icon-facebook"></i></a>
+          <a href="https://twitter.com/signup"><i class="fab fa-twitter icon-twitter ms-3 me-5"></i></a>
+          <a href="https://www.englishbolo.com/user-login.do" class="text-white top-nav-item">English</a>
+          <a href="#" class="text-white top-nav-item ms-4">MyPage</a>
+          <a href="#" class="text-white top-nav-item ms-4">Logout</a>
+        </div>
+    </div>
+    <div class="d-flex justify-content-between bg-white bottom-nav-bar py-2 px-5">
+      <div class="d-flex flex-row text-wrap justify-content-center logo">
+        <a href="./index.html">Gambit's<br><span class="text-center" id="arena">Arena</span></a>
+      </div>
+      <div class="d-flex justify-content-between align-items-center links-container">
+        <a href="./index.html" class="items fw-bold">Home</a>
+        <a href="https://g.co/kgs/LJBMP1" class="items fw-bold">Teams</a>
+        <a href="https://g.co/kgs/LJBMP1" class="items fw-bold">Players</a>
+        <a href="https://g.co/kgs/LJBMP1" class="items fw-bold">Fixtures</a>
+        <a href="https://g.co/kgs/LJBMP1" class="items fw-bold">Tables</a>
+        <a href="https://g.co/kgs/LJBMP1" class="items orange-border py-2 px-3 fw-bold">Latest News</a>
+      </div>
+    </div>
+  </nav>
+  <section class="container-fluid page-landing">
+    <div class="mx-3 mx-md-5 px-md-0"> 
+      <div class="row color me-0 p-md-0">
+        <nav class="d-md-none col-12 fixed-top nav-bar">
+          <a href="#" id="hamburger"><i class="fa fa-bars"></i></a>
+        </nav>
+        <div class="row pe-0 menu-hidden gap-5" id="mobile-menu">
+          <a href="#" class="col-12 d-flex justify-content-end text-decoration-none pe-3" 
+          id="close-icon"><i class="far fa-times-circle close"></i></a>
+          <a href="./about.html" class="menu-link text-decoration-none">About</a>
```

Comment:
- [ ] You should change this link to the home page (./index.html) so that the Home page link appears in the hamburger button dropdown while you are on the about page in mobile view. Kindly fix this.  

![Screenshot from 2021-09-10 16-24-56](https://user-images.githubusercontent.com/64894927/132839890-eaf81cb2-1f5a-446d-af03-5d1f33539e7a.png)

Reply:
Hi @henatan99, I have implemented the changes you requested for this project. There is a `Home` link to the **About** page section for the mobile version. 
![127 0 0 1_5500_about html](https://user-images.githubusercontent.com/77106962/132849592-deba31fa-5ae3-4d29-b1ff-9a1c922e85e1.png)

Reply:
Thanks for fixing this issue.